packages/rancher-external-ip-webhook/package.yaml

```diff
@@ -0,0 +1,2 @@
+url: https://github.com/rancher/externalip-webhook/releases/download/v1.0.0/rancher-externalip-webhook-1.0.0.tgz
+version: 1.0.0
```

Comment:
Added new package.yaml. Did not include the Chart.yaml/values.yaml patches that were in 2.5.

Reply:
Why not ? We still need to switch to mirror versions and do name chaning: https://github.com/rancher/charts/blob/dev-v2.5/packages/rancher-external-ip-webhook/generated-changes/patch/Chart.yaml.patch
The reason for name changing is because this chart also exists in another spot for 2.4, and we have to support both chart names, so we named them differently.

Reply:
Fixed